cylc/flow/cfgspec/workflow.py

```diff
@@ -1959,8 +1959,15 @@ def upgrade_graph_section(cfg: Dict[str, Any], descr: str) -> None:
     # Parsec upgrader cannot do this type of move
     with contextlib.suppress(KeyError):
         if 'dependencies' in cfg['scheduling']:
-            msg_old = '[scheduling][dependencies][X]graph'
-            msg_new = '[scheduling][graph]X'
+            if 'graph' in cfg['scheduling']['dependencies']:
+                msg_old = '[scheduling][dependencies]graph'
+                msg_new = '[scheduling][graph]R1'
+                list_cp = False
+            else:
```

Comment:
What does the flow.cylc file have to look like for this to get triggered?

Reply:
```
[scheduling]
    [[dependencies]]
        [[[P1D]]]
            graph = foo => bar
```
for example